ui/components/src/index.ts

```diff
@@ -25,4 +25,4 @@ export * from './TimeRangeSelector';
 export * from './context/ChartsThemeProvider';
 export * from './utils';
 export * from './model';
-export * from './test';
+export * from './test-utils';
```

Comment:
This was exporting the entire `tests` folder before into our public API, which was causing test-only libraries like `@testing-library/react` to appear in output bundles.